MIGRATION-TO-SCK.md

```diff
@@ -0,0 +1,322 @@
+# Migration from Splunk Connect for Kubernetes
+
+## Assumptions
+
+You are running Splunk Connect for Kubernetes(SCK) 1.4.9 and want to migrate to Splunk OpenTelemetry Connector for Kubernetes(SCK-OTEL) 1.0 GA. Previous versions of SCK are out of scope, but this guide is applicable for previous versions. You might encounter some version specific issues (such as missing/changed config options) but that shouldn't prevent you from proceeding with migration.
+
+## Changes in Components
+
+Splunk Connect for Kubernetes(SCK) has 3 components/applications:
+1.  Logs, metrics and traces from a kubernetes cluster (deployed as a DaemonSet)
```

Comment:
"1. Logs, metrics, and traces from a Kubernetes cluster..."
"2. An application to fetch cluster metrics from a Kubernetes cluster ..."
"3. An application to fetch Kubernetes object metadata from a Kubernetes cluster ..."